kits/cpp/README.md

```diff
@@ -14,6 +14,20 @@ This is the C++ implementation of the Python kit. It *should* have feature parit
   - [3.3 Actions](#33-actions)
   - [3.4 Additional features](#34-additional-features)
 
+## Getting Started
+
+To get started, download this folder from this repository.
+
+Your core agent code will go into `agent.py`, and you can create and use more files to help you as well. You should leave `main.py` alone as that code enables your agent to compete against other agents locally and on Kaggle.
+
+To quickly test run your agent, run
```

Comment:
This only works after they have built their agent. This may be helpful for some who don't have that much experience.

Reply:
Oh maybe add a comment there about running compile first?